app/Nova/Signature.php

```diff
@@ -147,14 +144,6 @@ public function fields(Request $request): array
             ]),
 
             self::metadataPanel(),
-
-            Hidden::make('Complete')
```

Comment:
Is there an issue open to replace this?

Reply:
No, but I don't think we need one. Almost no one uses this workflow and I want to completely replace it with DocuSign this summer anyway.